packages/electron/package.json

```diff
@@ -29,7 +29,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "theiaExtensions": [],
+  "theiaExtensions": [ ],
```

Comment:
Accidental change?

Reply:
The json formatter inserts the space upon save.